tests/_support/Page/Acceptance/Administrator/MenuManagerPage.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+class MenuManagerPage
+{
+    // include url of current page
+    public static $url = 'administrator/index.php?option=com_menus&view=item&layout=edit&menutype=mainmenu';
+
+    public static $menuFieldTitle = ['id' => 'jform_title'];
+
+    public static $selectMenutype = ['class' =>'btn-primary'];
+
+    public static $selectMenuTypeArticle = ['link' => 'Articles'];
+
+    public static $singleArticle = ['xpath' => "//div[@id='collapseTypes']//a[contains(text()[normalize-space()], 'Single Article')]"];
```

Comment:
When I am trying this xpath in firepath then it is working. But when I am using it here it's not working. Not getting what is problem. 

![xpath-issue](https://cloud.githubusercontent.com/assets/17275893/17337729/9f31149c-5901-11e6-9762-b8e005040183.png)